File: src/main/java/net/hectus/neobb/turn/default_game/flower/TAllium.java
package net.hectus.neobb.turn.default_game.flower;

import net.hectus.neobb.player.NeoPlayer;
import org.bukkit.block.Block;

public class TAllium extends FlowerTurn {
    public TAllium(NeoPlayer player) { super(player); }
    public TAllium(Block data, NeoPlayer player) { super(data, player); }

    @Override
    public int cost() {
        return 7;
    }

    @Override
    public void apply() {
        player.game.players().forEach(p -> p.inventory.removeRandom());
    }
}

File: src/main/kotlin/net/hectus/neobb/turn/default_game/flower/TAllium.kt
package net.hectus.neobb.turn.default_game.flower

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.player.NeoPlayer
import org.bukkit.block.Block

class TAllium(data: Block?, cord: Cord?, player: NeoPlayer?) : FlowerTurn(data, cord, player) {
    override val cost: Int = 7

    override fun apply() {
        player!!.game.players.forEach { it.inventory.removeRandom() }
    }
}
